### DISCUSSION TITLE
Differential accessibility test on batch corrected data?

### DISCUSSION THREAD

**Discussion by Thorged** [stuart-lab/signac#942] [Q&A] [answered]
Hello,

I have a multimodal dataset (2 samples, scRNA, scATAC on 10X platform), which I processed following the "Joint RNA and ATAC analysis: 10x multiomic" vignette. After seeing a strong batch seperation between my 2 samples in the joint UMAP visualization I found anchors using the RNA modality (SCTransformed). These I used to integrate the RNA data and the ATAC LSI embeddings. (This all worked great, thanks for the great and easy implementable tools!)

But now I am having some doubts how to correctly find differentially accessible peaks in the batch corrected data.
 Should I `IntegrateData()` my ATAC data to run `FindMarkers()` on? I understand that using an integrated peak matrix for differential testing is not recommended, as integration already introduces dependancies violating the statistical assumptions?
 My only other idea is to run DA on both samples independantly and compare the results/ create a common peak set as the **raw peak matrix suffers heavily from the batch effect**.

(Please excuse my inexperience, I just started working in the drylab and still try to wrap my head around things most probably consider common knowledge)

**Comment by timoast** [comment, marked as answer]
You shouldn't use batch corrected data for differential accessibility, instead you can provide batch as a latent variable in the model, eg `FindMarkers(..., latent.vars = 'orig.ident')` (setting whatever metadata variable separates the different batches). You could also run DA separately on each replicate and look at peaks that were detected across both.
